feflow/settings/integrators.py

```diff
@@ -0,0 +1,59 @@
+"""
+Settings objects for the different integrators used in the protocols.
+
+The reasoning is to have a base class that contains the settings that
+would be shared between different integrators, and subclasses of it
+for the specific integrator settings.
+"""
+
+try:
+    from pydantic.v1 import validator
+except ImportError:
+    from pydantic import validator  # type: ignore[assignment]
+
+from openff.units import unit
+from openff.models.types import FloatQuantity
+from gufe.settings import SettingsBaseModel
+
+
+class PeriodicNonequilibriumIntegratorSettings(SettingsBaseModel):
+    """Settings for the PeriodicNonequilibriumIntegrator"""
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    timestep: FloatQuantity["femtosecond"] = 4 * unit.femtoseconds
```

Comment:
Sorry very much a flyby 2 second review for now - the IntegratorSettings need to include information about the barostat frequency & possibly com removal, see: https://github.com/OpenFreeEnergy/openfe/blob/c4678745900e6425354407de5885c7c3eefd429b/openfe/protocols/openmm_utils/omm_settings.py#L195-L201

I'm not particularly sure it's the right design pattern, see: https://github.com/OpenFreeEnergy/openfe/issues/712, however because `get_system_generator` (https://github.com/OpenFreeEnergy/openfe/blob/main/openfe/protocols/openmm_utils/system_creation.py#L26) currently asks for an integrator settings object (and you're using it in the neq protocol), you end up needing that additional field.

Reply:
Ah maybe there's going to be a typing issue - I can see that maybe we have to have a discussion about this.

Reply:
The `barostat_frequency` doesn't seem to be needed or used, as far as I can tell, at least not in the NeqCycling protocol, which makes me wonder if this is something we would like to change. What are the scenarios where changing this frequency is desired?

Reply:
In the end it was indeed needed for "real-life" simulations, this probably means we need to add more testing that resembles this situation.